### ISSUE TITLE
Rename "Python" to "Python 2"

### ISSUE THREAD

**Issue by Naatan** [Komodo/KomodoEdit#1753]
Python 2 is simply called "Python" in Komodo, wheras Python 3 is called exactly that. This leads to users confusing "Python" with "Python 3". We should be explicit with our naming.

**Comment by mitchell-as** [comment]
This can have very far-reaching implications if we're not careful. I think we should only change the outward-facing name in the UI of "Python", rather than the internal name. Perhaps this is obvious, but I wanted to make sure.

**Comment by cgchoffman** [comment]
@mitchell-as thanks for the reminder.  I'll keep the changes cosmetic.  This needs to be applied in a few places:
- Run Command Interpolation string, only change the description in the UI
- Likely "debugging not configured" dialog
- Prefs
- Interactive Shell, Tools > Interactive Shell & Sidebar Button
- Template file
- Language dropdown.

**Comment by Naatan** [comment]
Yeah, sorry I only meant for this to be cosmetic. Basically like we did with the macro > usercript stuff.

**Comment by Defman21** [comment]
@cgchoffman also Documentation scope in Commando.
